system-x/services/azure/pom.xml

```diff
@@ -16,9 +16,13 @@
 
     <properties>
         <azure.clients.version>12.11.1</azure.clients.version>
+        <azure.event.hubs.version>5.12.0</azure.event.hubs.version>
+        <azure.core.version>1.28.0</azure.core.version>
     </properties>
 
     <modules>
+        <module>azure-common</module>
```

Comment:
please rename the folder to `common` only

Reply:
ok, renamed